streams/iq.m3u

```diff
@@ -5,14 +5,34 @@ http://rtmp1.abnsat.com/hls/arabic.m3u8
 https://5ab29cc78f681.streamlock.net/afarinTV/livestream/playlist.m3u8
 #EXTINF:-1 tvg-id="AlEtejah.iq",Al-Etejah (576p) [Not 24/7]
 http://streaming.aletejahtv.iq:1935/etejah-newLive/smil:etejahNew.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="AlghadeerTV.iq",AlGhadeer (720p) [Not 24/7]
-https://sd343444.vodu.store:3356/live/Alghadeer/index.m3u8
+#EXTINF:-1 tvg-id="AlforatTV.iq",Alforat TV (1080p) 
+http://95.216.180.111:1935/live/10/playlist.m3u8
+#EXTINF:-1 tvg-id="AlGhadTV.eg",Al Ghad (720p) [Not 24/7]
```

Comment:
Only iraq channels are added to iq.m3u. Please remove channels with a tvg-id that does not end with `.iq` and add them to the playlist for that country.

Reply:
Ok